app/src/main/res/layout/fragment_events.xml

```diff
@@ -99,6 +99,7 @@
                             android:id="@+id/eventsRecycler"
                             android:layout_width="match_parent"
                             android:layout_height="match_parent"
+                            android:paddingTop="@dimen/padding_medium"
```

Comment:
Why?

Reply:
ShimmerEvents have have space near top which disappears when Events get loaded so added padding for consistency. Wanted to open another PR, but pushed in this by mistake. Do you want me to remove this or is it okay?
![image](https://user-images.githubusercontent.com/37517284/52802358-cbe48900-30a5-11e9-9984-f8295c8293c7.png)

![image](https://user-images.githubusercontent.com/37517284/52802391-dd2d9580-30a5-11e9-84c1-6aed9256a081.png)

Reply:
Is this okay? @iamareebjamal